distribution/docs/src/main/resources/_contents/_developing/developing-intro-contents.adoc

```diff
@@ -170,100 +170,52 @@ ${ddf-branding-lowercase}${at-symbol}local>list | grep -i example
 
-* Be sure that only one feature in the `features.xml` has the `auto-install` tag.
+* Ensure only one feature in the `features.xml` has the same name as the project itself. (e.g. If the project name is `spatial-app`, there should be one feature named `spatial-app`)
+This is the feature that the installer displays to the user (name, description, version, etc.) and  provides the description of what the application contains.
+This feature should only declare the `prerequisite` features of the app and any bundles or features that are required by all of the features within the app.
+* Be sure all features that should be installed by default include the `auto-install` flag.
 +
 [source]
 ----
 install='auto'
 ----
 +
-This is the feature that the installer displays to the user (name, description, version, etc.). It is typically named after the application itself and the description provides a complete application description.
-* Be sure that the one feature specified to `auto-install` has a complete list of all of its dependencies in order to ensure the dependency tree can be constructed correctly.
+* Each feature should declare its `prerequisite` features. These are the features that are required to be installed prior to this feature starting up. At a minimum this should include the application feature, so that the application prerequisites and required dependencies are satisified before this feature starts.
++
+[source]
+----
+<feature prerequisite="true">feature1</feature>
+----
 
 ===== Auto-starting an Application Feature
 
-Within the `features.xml` file for an application, one feature will have the install attribute set to `auto`.
-Within this feature, refer to any dependencies of the application as well as any features that should start automatically.
+Within the `features.xml` file for an application, the feature that is named the same as the project will have the install attribute set to `auto`.
+Within this feature, refer to any prerequisites of the application as well as any features or bundles within the application that are required by all features.
+Features that should start automatically should have install set to `auto`.
 Other features should have install set to `manual`.
 
 The following example demonstrates configuring features to be auto-started. The naming convention for this feature is typically "`application name`" + "`-app`," as shown.
 
 .Auto-start Features
 [source,xml,linenums]
 ----
-<feature name="catalog-app" install="auto">
-    <feature>platform-app</feature>
-    <feature>catalog-core</feature>
-    <feature>catalog-core-metricsplugin</feature>
-    <feature>catalog-core-sourcemetricsplugin</feature>
-    <feature>catalog-transformer-thumbnail</feature>
+<feature name="example-app" install="auto">
+    <feature prerequisite="true">prereq-feature</feature>
+    <bundle>mvn:com.example/example/dependency</bundle>
```

Comment:
The format for your sample `<bundle>`s is wrong. It should be
`mvn:[group name]/[artifact name]/[version][/[optional classifier]]`

e.g.
`<bundle>mvn:com.example/example.dependency/1.3</bundle>`

or in the less common case where a classifier is specified:
`<bundle>mvn:com.example/example.dependency/1.3/war</bundle>`

(Assuming an artifact name of `example.dependency`.)